### PR TITLE
Updated changelog v0.6.2

### PR DESCRIPTION
**Pull request by PacoDw**
## Description

Updated the changelog to v0.6.2

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [x] Documentation fix/enhancement

## Required Checklist:

- [x] I have signed the MongoDB CLA
- [x] I have read the Terraform contribution guidelines 
- [x] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirments
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run make fmt and formatted my code

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,25 @@
...
+- Added golangci configuration and travis fix [\#248](https://github.com/terraform-providers/terraform-provider-mongodbatlas/pull/248) ([PacoDw](https://github.com/PacoDw))
+
+
+**Fixed bugs:**
+
+- Adding 16 whitelist entries at the same time causes an error [\#252](https://github.com/terraform-providers/terraform-provider-mongodbatlas/issues/252)
+- Cannot re-apply config when M2/M5 `disk\_size\_gb` is specified incorrectly [\#115](https://github.com/terraform-providers/terraform-provider-mongodbatlas/issues/115)
+- accepter\_region\_name not required for AWS on read/import/update [\#53](https://github.com/terraform-providers/terraform-provider-mongodbatlas/issues/53)
+- Fix resource/project\_ip\_whitelist - modify ip whitelist entry validation and fix acctests [\#257](https://github.com/terraform-providers/terraform-provider-mongodbatlas/pull/257) ([marinsalinas](https://github.com/marinsalinas))
+
```
Comment (themantissa) [inline CHANGELOG.md:17]: Missing 243: https://github.com/terraform-providers/terraform-provider-mongodbatlas/issues/243
We have now fixed two bugs w/ that issue but this should close it again (next time I'll be sure to open a new issue when we know unrelated)

**Review by themantissa — CHANGES_REQUESTED** [review]
Just one issue missing I think, otherwise lgtm.

File: CHANGELOG.md (hunk 1 above)
Comment (PacoDw) [reply CHANGELOG.md:17]: done, thanks

**Review by themantissa — APPROVED** [review]
LGTM!

**Review by coderGo93 — APPROVED** [review]
LGTM

**Pull request merged**